x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/alert_summary/rule_alerts_summary.tsx

```diff
@@ -142,8 +133,12 @@ export const RuleAlertsSummary = ({ rule, filteredRuleTypes }: RuleAlertsSummary
           </EuiTitle>
         </EuiFlexItem>
       </EuiFlexGroup>
-      <Chart size={['100%', '35%']}>
-        <Settings tooltip={TooltipType.None} theme={theme} />
+      <EuiSpacer size="m" />
+      <Chart size={{ height: 50 }}>
+        <Settings
+          tooltip={TooltipType.None}
+          theme={[EUI_SPARKLINE_THEME_PARTIAL, EUI_CHARTS_THEME_LIGHT.theme]}
```

Comment:
Shouldn't we check the dark and light theme similar to this:
https://github.com/elastic/kibana/blob/cb3e0ec67a0a20767204be752c31969bd8a3ff61/x-pack/plugins/infra/public/pages/logs/log_entry_categories/sections/top_categories/single_metric_sparkline.tsx#L43

Reply:
@maryam-saeidi  👍🏻